docs/index.md

```diff
@@ -1117,9 +1117,42 @@ app/components
 </div>
 ```
 
+##### Translations
+
+In order to support `I18n` translations scoped to the component you can add a dynamic translations loader in `config/locale/components.rb`.
+
+```ruby
+require "view_component/i18n"
+ViewComponent::I18n.load components_root: "#{Rails.root}/app/components", glob: "**/*component.yml"
+```
+
+This allows for adding a "sidecar" YAML file that will define translations
+scoped to the component, e.g. `app/components/greeter/component.yml`:
```

Comment:
```suggestion
scoped to the component, e.g. `app/components/greeting/component.yml`:
```

Reply:
What about using `example` as the component name to stay consistent with the rest of the docs?